modules/learn/pages/clusters-and-availability/automatic-failover.adoc

```diff
@@ -136,7 +177,7 @@ If node #4 becomes unresponsive, auto-failover can be triggered, since prior to
 However, if instead, node #5 becomes unresponsive, auto-failover is not triggered; since prior to unavailability, the Analytics Service was running only on one node (#5), which is below the auto-failover policy requirement for the Analytics Service (which is 2).
 
 [#data-service-preference]
-* A cluster has the following three nodes:
+* A cluster has the following two nodes:
```

Comment:
It should be three nodes -- the original sentence was correct -- even though we are changing the description of one of the nodes (to an arbiter node), there are still 3 nodes in the cluster.  If there weren't three nodes, a quorum could not be formed after one node fails.  So, saying "A cluster has the following **two** nodes:" is pretty confusing.

* A cluster has the following three nodes: